curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-oop-by-building-a-platformer-game/64acf287857bb38e6dd7ca69.md

```diff
@@ -15,7 +15,7 @@ You need to use the `proportionalSize` function here to make sure the player's `
 
 # --hints--
 
-You should have an `if` statement that checks if the right key was pressed and the player's `x` position is less than `proportionalSize(400)`.
+You should have an `if` statement that checks if the right key was pressed and the player's `x` position is less than `proportionalSize(400)`.The `this` keyword should not be used here because that is only for the `Player` class and not the use of the `player` object.
```

Comment:
Let's update this last sentence to tweak the grammar and phrase it as a general reminder of the `this` keyword.

```suggestion
You should have an `if` statement that checks if the right key was pressed and the player's `x` position is less than `proportionalSize(400)`.  Remember that the `this` keyword should not be used here because that is only for the `Player` class and not for the `player` object.
```